src/__test__/__snapshots__/Form.spec.js.snap

```diff
@@ -0,0 +1,74 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Form renders a basic form 1`] = `
+<form
+  className=""
+  onSubmit={[Function]}
+  style={
+    Object {
+      "backgroundColor": "#FFFFFF",
+      "display": "grid",
+      "gridTemplateColumns": "100%",
+      "gridTemplateRows": "minmax(500px,min-content)",
+      "justifyContent": "center",
+      "maxWidth": "100%",
+      "width": "500px",
+    }
+  }
+>
+  <div
+    style={
+      Object {
+        "alignItems": undefined,
```

Comment:
i don't think these undefineds and NaNs should be expected, i think they'll be actual values if the object being passed into the component contains these fields